src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1687,37 +1765,36 @@ subroutine apply_calving_velocity(meshPool, geometryPool, velocityPool, domain,
       call mpas_dmpar_field_halo_exch(domain, 'requiredCalvingVolumeDynEdge')
       call mpas_timer_stop("halo updates")
 
-      ! c. Now calculate calvingThickness for each cell
+      ! c. Now apply ablation to each cell
       do iCell = 1, nCellsSolve
          if ((li_mask_is_dynamic_ice(cellMask(iCell))) .and. (bedTopography(iCell) <= config_sea_level)) then
             ! Can loop over all dyn cells - only ones with calving on their edges will have calving applied.
             do iNeighbor = 1, nEdgesOnCell(iCell)
                iEdge = edgesOnCell(iNeighbor, iCell)
                ! No need to check what type of edge this is - only required edges are nonzero
-               requiredCalvingVolumeDynCell(iCell) = requiredCalvingVolumeDynCell(iCell) + &
-                          requiredCalvingVolumeDynEdge(iEdge) ! Keep running total
+               requiredAblationVolumeDynCell(iCell) = requiredAblationVolumeDynCell(iCell) + &
+                          requiredAblationVolumeDynEdge(iEdge) ! Keep running total
             enddo
             ! c. Apply calving here
-            removeVolumeHere = min(cellVolume(iCell), requiredCalvingVolumeDynCell(iCell))  ! Don't use more than available
-            calvingThickness(iCell) = removeVolumeHere / areaCell(iCell)
-            calvedVolumeDynCell(iCell) = removeVolumeHere
```

Comment:
The corresponding rename of this line is missing.

Reply:
Added back